Sources/FluentPostgresDriver/FluentPostgresDatabase.swift

```diff
@@ -65,6 +67,7 @@ extension _FluentPostgresDatabase: Database {
             for create in e.createCases {
                 _ = builder.add(value: create)
             }
+            self.logger.debug("\(builder.query)")
```

Comment:
Should this not be logged for `.delete` and `.create` as well?

Reply:
Yes it should! Good catch!